### PR TITLE
ui vs code

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-14-UI vs Code for Defining Interfaces.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+status: publish
+categories: [Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+## UI vs Code
+
+The day of defining your UIs in a different place from your code and later on connecting it to the code are about to an end. The writing has been on the wall for a long time, but lately more and more platforms are embracing this paradigm shift.
```
Comment (MsSlacker) [inline _posts/2019-11-14-UI vs Code for Defining Interfaces.md]: day - > days

Comment (MsSlacker) [reply _posts/2019-11-14-UI vs Code for Defining Interfaces.md]: about to come to an end

File: _posts/2019-11-14-UI vs Code for Defining Interfaces.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+With declarative programming and new frameworks like React Native, Flutter and now SwiftUI we see most major players moving out of xml based interfaces towards befining both your interface and your data binding in code in one single place. Still waiting for Native Android and Windows to get the memo.
+
+The benefits are plentiful, the major one being having a single source of truth for all your interfaces and clearly see how its data flow works and how it reacts with different data from one place. You might still need to open more than one file to see the whole tree, but you can get a generic understanding of how the screen works from one single file.
```
Comment (MsSlacker) [inline _posts/2019-11-14-UI vs Code for Defining Interfaces.md:20]: being having sounds so weird....

Comment (MsSlacker) [reply _posts/2019-11-14-UI vs Code for Defining Interfaces.md:20]: see - > seeing or being able to see

Comment (MsSlacker) [reply _posts/2019-11-14-UI vs Code for Defining Interfaces.md:20]: Pretty short :P

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-14-UI vs Code for Defining Interfaces.md (hunk 2 above)
Comment (valeIT) [reply _posts/2019-11-14-UI vs Code for Defining Interfaces.md:20]: Mostly talked about the topic already from different angles so didn't want to repeat those points too. but would be nice to link those as well

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**